README.md

```diff
@@ -64,7 +89,14 @@ argument. Finally, demonstrate calling `doSomething` with a function.**
 
 Your Answer:
 ```js
-// write code here
+var doSomething = function(thingToDo) {
+  return thingToDo;
+}
+
+function(){
+  doSomething()
+}
+;
 ```
 
```

Comment:
Close. Couple things...

* `doSomething` should actually run the function that is passed in as an argument (`thingToDo), not return it.
* `doSomething` does not need to be called inside of an anonymous function. Actually, in your example, `doSomething` is never run because that anonymous function is never executed.